src/main/kotlin/no/nav/syfo/persistering/api/SendRegisterPapirSykmeldingOppgaveRest.kt

```diff
@@ -207,11 +208,40 @@ fun Route.sendPapirSykmeldingManuellOppgave(
                                     }
                                 }
                                 Status.MANUAL_PROCESSING -> {
-                                    log.info(
-                                        "Ferdigstilling av papirsykmeldinger manuell registering traff regel MANUAL_PROCESSING {}",
-                                        StructuredArguments.keyValue("oppgaveId", oppgaveId)
-                                    )
-                                    call.respond(HttpStatusCode.BadRequest, validationResult)
+
+                                    // Sjekk om reglene som slo til er hvitelistet i WhitelistedRuleHits
+                                    if (validationResult.ruleHits.isAllRulesWhitelisted()) {
```

Comment:
her gjør vi en del av de samme tingene for både OK og Manuell prosessering. Hvis vi snur om litt på sjekkene slipper vi å duplisere så mye. Men det er ikke krise altså, og det er jo lett å lese sånn som dette også :)

Reply:
kanskje ryddigere slik du foreslår. gjør et forsøk